### DISCUSSION TITLE
Wild increase in total usage during disconnections.

### DISCUSSION THREAD

**Discussion by Wheemer** [emporia-vue-local/esphome#140] [Q&A]
Hello;

On day two of flashing this firmware. Yesterday I have a huge usage report during a time when I was changing up routers and fixing the wifi, passed it off as maybe rare. Today I disconnected the power for 20 mins and there's a huge incorrect spike in usage again.

Is there any sort of buffer or any way to catch these so they don't throw all my data out the window every time I have wifi issues?

Thanks
Steve

**Comment by MarkoPaasila** [comment]
add `method: left` to the [Total Daily Energy](https://esphome.io/components/sensor/total_daily_energy.html) sensors.

**Comment by Wheemer** [comment]
Hello;

Something like:

`   power_id: total_power
    method: left
    accuracy_decimals: 2`

Since this would be applying to all the sensors, is there any downfall to this? Why isn't it set with this is the suggested config?

**Comment by MarkoPaasila** [comment]
Yes. remember commas :-). No this will apply only to the sensor where you apply it. Why it isn't default I don't know.
